app/models/ability.rb

```diff
@@ -51,6 +54,7 @@ def initialize(user)
       can [:update, :read, :read_billing_information], Provider, symbol: user.provider_id.upcase
       can [:manage], ProviderPrefix, provider_id: user.provider_id
       can [:manage], Client, provider_id: user.provider_id
+      cannot [:transfer], Client, provider_id: user.provider_id
```

Comment:
Why do we need to explicitly not allow?

Reply:
to not allow "provider_admins" to have the ability to execute `transfer` but have access to any of the other methods (cancan is not limited to crud methods from what i gather). 

actually the condition `provider_id: user.provider_id` is not necessary.